module/VuFind/src/VuFind/Db/Service/RatingsServiceInterface.php

```diff
@@ -79,6 +79,16 @@ public function getCountsForRecord(
      */
     public function deleteByUser(UserEntityInterface|int $userOrId): void;
 
+    /**
+     * Deletes ratings by given rating ids and user id
```

Comment:
Add punctuation for consistency.

```suggestion
     * Deletes ratings by given rating ids and user id.
```